app/models/manageiq/providers/openstack/cloud_manager/vm/associate_ip.rb

```diff
@@ -0,0 +1,47 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::AssociateIp
+  extend ActiveSupport::Concern
+
+  included do
+    supports :associate_floating_ip do
+      if cloud_tenant.nil? || Rbac.filtered(cloud_tenant.floating_ips).empty?
```

Comment:
@mansam @kbrock One would add a model to RBAC if there is 1) a requirement to limit the objects a uses can see in the class, 2) the objects in the class are directly accessible in the UI - meaning a user does not have to navigate to them through a parent object that already does RBAC (like Vm, as @kbrock mentioned)

It seems that is that if these floating_ip object are only accessible through Vms then it would not be necessary to explicitly do RBAC on them